darwin/dataset/utils.py

```diff
@@ -93,7 +100,10 @@ def extract_classes(annotations_path: Path, annotation_type: str) -> Tuple[Dict[
             continue
 
         for annotation in annotation_file.annotations:
-            if annotation.annotation_class.annotation_type != annotation_type:
+            if (
```

Comment:
I believe this may be inconsistent with [this logic](https://github.com/v7labs/darwin-py/blob/0f799a589c884cebd699fb62b2713758f4031464/darwin/dataset/local_dataset.py#L328). Here it seems "complex_polygon" will not be considered in creating the class list, while in the linked logic "complex_polygon" is considered for an annotation.annotation_class.annotation_type.

Reply:
Not sure which is better: remove complex_polygon from linked logic or add it here

Reply:
Great input, I have not changed anything considering that logic. So the behavior should be the same as main darwin-py in that regard. BUT, that does not stop us from fixing it if there is an issue.

Reply:
I suggest, we skip this for now. It might be a bug, but it would be a pre-existing condition and we have the same behavior as before. Besides, I think that complex-polygons are getting deprecated. 

We could raise a bug report thought.